scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -160,15 +128,56 @@ private void changeZValue(){
     double zValue = zValueSlider.getValue();
```

Comment:
I can't comment on line 126, but I just noticed that there should be an extra space between the parenthesis and curly brackets

```java
private void changeZValue() {

```